Util/openl-opentelemetry-javaagent-extension/src/org/openl/opentelemetry/javaagent/extension/MethodSingletons.java

```diff
@@ -0,0 +1,34 @@
+package org.openl.opentelemetry.javaagent.extension;
+
+import io.opentelemetry.api.GlobalOpenTelemetry;
+import io.opentelemetry.instrumentation.api.instrumenter.Instrumenter;
+import io.opentelemetry.instrumentation.api.instrumenter.SpanKindExtractor;
+import io.opentelemetry.instrumentation.api.instrumenter.code.CodeAttributesExtractor;
+import io.opentelemetry.instrumentation.api.instrumenter.code.CodeAttributesGetter;
+import io.opentelemetry.instrumentation.api.instrumenter.code.CodeSpanNameExtractor;
+import io.opentelemetry.instrumentation.api.instrumenter.util.ClassAndMethod;
+
+public final class MethodSingletons {
+    private static final String INSTRUMENTATION_NAME = "io.opentelemetry.methods";
```

Comment:
What this constant mean? Why this constant has `"io.opentelemetry.methods"` value? Why not `"org.openl.rules"`? Is it requirement of the OpenTelemetry (provide proof link)?